<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string>
     QWidget{
        background:#0f172a;
     }
     #panel{
        background:#111827;
        border-radius:18px;
     }
     QPushButton{
        background:#1f2937;
        color:#d1d5db;
        border-radius:12px;
        font-size:18px;
        padding:6px;
     }
     QPushButton:hover{
        background:#334155;
     }
     QPushButton[class=&quot;op&quot;]{
        background:#1d4ed8;
        color:white;
     }
     QPushButton[class=&quot;eq&quot;]{
        background:#3b82f6;
        color:white;
        font-weight:600;
     }
     QLabel{
        color:white;
     }
    </string>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="result">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>87</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string>font-size:36px;font-weight:700;</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="grid">
        <item row="1" column="2">
         <widget class="QPushButton" name="nine">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="four">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="six">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="five">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="one">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="eight">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="seven">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="zero">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="three">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="two">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="equal">
          <property name="text">
           <string>=</string>
          </property>
          <property name="class" stdset="0">
           <string>eq</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="delete_all">
          <property name="text">
           <string>AC</string>
          </property>
          <property name="class" stdset="0">
           <string>op</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="plus">
          <property name="text">
           <string>+</string>
          </property>
          <property name="class" stdset="0">
           <string>op</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="minus">
          <property name="text">
           <string>-</string>
          </property>
          <property name="class" stdset="0">
           <string>op</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="mutiply">
          <property name="text">
           <string>ร</string>
          </property>
          <property name="class" stdset="0">
           <string>op</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="divide">
          <property name="text">
           <string>รท</string>
          </property>
          <property name="class" stdset="0">
           <string>op</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
